docs/content/direct/get-started.md

```diff
@@ -1,6 +1,30 @@
 # Getting Started with KubeStellar
 
-This page shows one concrete example of steps 2--7 from the [full Installation and Usage outline](user-guide-intro.md#the-full-story). This example produces a simple single-host system suitable for kicking the tires, using [kind](https://kind.sigs.k8s.io/) to create three new clusters to serve as your KubeFlex hosting cluster and two WECs. This page concludes with forwarding you to one example of the remaining steps.
+There are multiple ways to get started with Kubestellar, reflecting its flexible architecture that supports various deployment patterns. This page presents two approaches:
```

Comment:
I think it would be good to be a little more specific about the scope of the script and its relationship with the step-by-step. I might say things like:
- The script does the same things as the parts of the step-by-step from the checking of prerequisites through creating and registering two WECs.
- The other remarks below apply equally, including the establishing of the prerequisites (including making `kind` able to create three clusters) and the remarks about variables to set and what to do afterwards.
- The script does things in a different order, using as much concurrency as possible to get as much speed as possible. This makes it more complicated than the step-by-step but only for technical reasons related to the concurrency.